internal/panda_controller/docker_grpc_agent.go

```diff
@@ -192,3 +193,32 @@ func (pa *dockerGrpcPandaAgent) stopContainer(ctx context.Context) error {
 
 	return nil
 }
+
+func copyFileHelper(file_path string, shared_dir string) (int64, error) {
```

Comment:
file_path and shared_dir could have better descriptors such as source_file_path or dest_shared_dir. In addition, a comment that says what the function does and what each parameter is would be nice.